lib/live_debugger/services/garbage_collector/gen_servers/garbage_collector.ex

```diff
@@ -4,13 +4,16 @@ defmodule LiveDebugger.Services.GarbageCollector.GenServers.GarbageCollector do
   """
 
   use GenServer
+  alias LiveDebugger.API.SettingsStorage
+  alias LiveDebugger.App.Events.UserChangedSettings
 
   alias LiveDebugger.Services.GarbageCollector.GenServers.TableWatcher
 
   alias LiveDebugger.Services.GarbageCollector.Actions.GarbageCollecting,
     as: GarbageCollectingActions
 
   alias LiveDebugger.Bus
+
   alias LiveDebugger.Services.GarbageCollector.Events.GarbageCollected
```

Comment:
When aliasing try to group them logically so it's easier to read
```suggestion
  use GenServer
  
  alias LiveDebugger.API.SettingsStorage
  alias LiveDebugger.Services.GarbageCollector.GenServers.TableWatcher

  alias LiveDebugger.Services.GarbageCollector.Actions.GarbageCollecting,
    as: GarbageCollectingActions

  alias LiveDebugger.Bus
  alias LiveDebugger.Services.GarbageCollector.Events.GarbageCollected
  alias LiveDebugger.App.Events.UserChangedSettings
```